rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -7609,6 +7609,84 @@ public void getQueuedPegoutsCount_after_RSKIP271_activation() throws IOException
         assertEquals(2, bridgeSupport.getQueuedPegoutsCount());
     }
 
+    @Test
+    public void test_processPegoutsInBatch_before_RSKIP271_activation() throws IOException {
```

Comment:
what are we testing here? I mean, the `ReleaseRequestQueue` is empty so the RSKIP activation is irrelevant, right?

Reply:
Resolved

Reply:
Resolved